controllers/suite_test.go

```diff
@@ -72,36 +70,15 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(err).ToNot(HaveOccurred())
 	Expect(cfg).ToNot(BeNil())
 
-	err = syncv1alpha1.AddToScheme(scheme.Scheme)
-	err = argov1alpha1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
+	Expect(syncv1alpha1.AddToScheme(scheme.Scheme)).To(Succeed())
+	Expect(argov1alpha1.AddToScheme(scheme.Scheme)).To(Succeed())
 
 	// +kubebuilder:scaffold:scheme
```

Comment:
Is kubebuilder scaffolding going to regenerate some of the things that we have removed here?

Reply:
Not that I'm aware of because I added to it manually